lib/ansible/modules/dnf5.py

```diff
@@ -357,9 +357,11 @@ def is_installed(base, spec):
 
 
 def is_newer_version_installed(base, spec):
+    if "/" in spec:
+        spec = spec.split("/")[-1][:-4]
```

Comment:
Should we check it ends with `.rpm` before snipping the last 4 characters?

Reply:
I guess it's not necessary, since we know it's a path at this point.

Reply:
I added the check anyway, maybe it will be more obvious what `[:-4]` means :-)

I was hoping to just use `spec.removesuffix(".rpm")` that was added in Python 3.9 but Python versions down to 3.6 can be installed from Fedora repositories on Fedora 38. I don't know if that will be the case on Fedora 41 where dnf5 is supposed to be the default package manager but didn't use `removesuffix` just to be safe anyway.

Reply:
Ah, and it would fail for `microdnf` on F38, so yeah, can't use `removesuffix`...

Reply:
Looks good, thanks!